src/timing.c

```diff
@@ -16,29 +16,14 @@ jl_module_t *jl_module_root(jl_module_t *m);
 extern "C" {
 #endif
 
-JL_DLLEXPORT int jl_tracy_enabled(void) {
-#ifdef USE_TRACY
-    return 1;
-#else
-    return 0;
-#endif
-}
-
-JL_DLLEXPORT int jl_ittapi_enabled(void) {
-#ifdef USE_ITTAPI
+JL_DLLEXPORT int jl_timings_enabled(void) {
```

Comment:
```suggestion
JL_DLLEXPORT int jl_timing_enabled(void) {
```

The macro is a bit inconsistent with this, but `jl_timing_` is what we use everywhere else